README.md

```diff
@@ -84,6 +84,17 @@ Delete cookie:
 Cookies.remove('name');
 ```
 
+Please notice that removing unexisting cookie does not raise any exception nor return any value
```

Comment:
That doesn't mean the cookie doesn't exist, that means the cookie is not accessible. It is impossible to verify if the cookie exist or not.

We have discussed this in the past, see https://github.com/carhartl/jquery-cookie/issues/164#issuecomment-19003359.